src/raftstore/store/metrics.rs

```diff
@@ -221,4 +221,10 @@ lazy_static! {
             &["type"],
             exponential_buckets(0.001, 1.59, 20).unwrap() // max 10s
         ).unwrap();
+
+    pub static ref KEY_NOT_IN_REGION: IntCounter = register_int_counter!(
```

Comment:
do we need to add a label so we can locate where it raises?

Reply:
Maybe not a good idea. Constant labels are preferred. Dynamic labels are likely to cause memory never being released because these metric instance need to be kept permanently.